modules/ROOT/pages/database-management.adoc

```diff
@@ -83,8 +83,8 @@ The table at <<tbl-tomb-purge>> shows when tombstones are purged automatically a
 |Automatic
 |Tombstones are not automatically purged from the bucket.
```

Comment:
enable_shared_bucket_access is now always on, so you could remove this, or say that 

Sync Gateway automatically prunes tombstones